src/libstd/process.rs

```diff
@@ -233,8 +233,9 @@ impl fmt::Debug for ChildStdout {
     }
 }
 
-/// A handle to a child process's stderr. This struct is used in the [`stderr`]
-/// field on [`Child`].
+/// A handle to a child process's stderr. It can be used to
```

Comment:
I'm not saying to remove it, I'm saying that there needs to be whitespace between.